src/structures/GuildChannel.js

```diff
@@ -388,7 +388,7 @@ class GuildChannel extends Channel {
   setParent(channel, { lockPermissions = true, reason } = {}) {
     return this.edit(
       {
-        parentId: channel?.id ?? channel ?? null,
+        parent: channel?.id ?? channel ?? null,
```

Comment:
```suggestion
        parent: channel ?? null,
```
i suppose this can also be changed (as that was originally what was on the PR)

Reply:
Yeah I guess since we're resolving the id anyway in .edit might as well keep that.

Reply:
Yes please. This check is already in the edit, so it would be duplicating the check.